core/services/relay/evm/mercury/wsrpc/client_test.go

```diff
@@ -176,3 +176,65 @@ func Test_Client_LatestReport(t *testing.T) {
 		})
 	}
 }
+
+func Test_Client_RawLatestReport(t *testing.T) {
```

Comment:
@samsondav I borrowed this test from Test_Client_Transmit. 

I thought about just refactoring the tests and creating 1 for handleTimeout so we don't need to have this tested twice